fixtures/revocationList.json

```diff
@@ -18,9 +18,9 @@
     },
     "proof": {
       "type": "Ed25519Signature2018",
```

Comment:
Wondering what folks feelings are for switching to Ed25519Signature2020? We're all speed ahead at DB for that change because it switches to using multibase, which gets good compression in CBOR-LD, and because it makes a clean break from jws, which is what JsonWebSignature2020 is for.